gst/nnstreamer/tensor_source/tensor_src_iio.c

```diff
@@ -52,6 +52,7 @@
 #include <gst/gst.h>
 #include <glib.h>
 #include <string.h>
+#include <stdio.h>
```

Comment:
Please do not add unrelated changes.
The topic of this commit is ```[IIO] Added doxygen entries ```

Reply:
my mistake
updated with changes applied to appropriate commit.